bin/build_container_image

```diff
@@ -12,7 +16,7 @@ set -e
 
 docker build . -t $IMAGE_NAME
 
-[[ $TRAVIS_PULL_REQUEST != "false" ]] && exit 0
+[[ -z "$REGISTRY_USERNAME" ]] && exit 0
```

Comment:
I thought it simpler to just check for the presence of the REGISTRY_USERNAME, as opposed to checking for anything CI specific